crates/bevy_ecs/src/component/mod.rs

```diff
@@ -26,9 +26,25 @@ use thiserror::Error;
 pub trait Component: Send + Sync + 'static {}
 impl<T: Send + Sync + 'static> Component for T {}
 
+/// The storage used for a specific component type.
+///
+/// # Examples
+/// The [`StorageType`] for a component is normally configured via `World::register_component`.
+///
+/// ```
+/// # use bevy_ecs::{prelude::*, component::*};
+///
+/// struct A;
+///
+/// let mut world = World::default();
+/// world.register_component(ComponentDescriptor::new::<A>(StorageType::SparseSet));
+/// ```
 #[derive(Debug, Copy, Clone, Eq, PartialEq)]
 pub enum StorageType {
+    /// Provides fast and cache friendly iteration, but slower adding and removing of components.
```

Comment:
more an English question than a review: should it be fast/slow or faster/slower, or is it ok to mix fast/slower like that?

Reply:
It doesn't parse as incorrect in its current form to me, as a native English speaker.

Nits though:
1. I would probably hyphenate "cache-friendly".
2. I would use addition and removal instead of adding and removing, since you really want a noun there.

Reply:
I actually just copied and pasted what was in the `bevy_ecs` README haha.
However I agree I think addition/removal fits better :)